src/rules/no-compositing-layer-props.ts

```
@@ -0,0 +1,132 @@
+        value.includes('translate3d') ||
+        value.includes('scale3d') ||
+        value.includes('translateZ');
+    }
+
+    function checkProperty(propertyName: string, propertyValue?: string): boolean {
+      const normalizedName = normalizePropertyName(propertyName);
+      if (COMPOSITING_PROPERTIES.has(normalizedName)) {
+        // Special case for opacity - only warn if it's animated or fractional
+        if (normalizedName === 'opacity') {
+          if (!propertyValue) return false;
+          const numValue = parseFloat(propertyValue);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)